lakefs_provider/operators/with_metadata_operator.py

```diff
@@ -47,6 +48,9 @@ def _get_current_dag_dict(self, context: Dict[str, Any]) -> Dict[str, Any]:
         return ret
 
     def enrich_metadata(self, context: Dict[str, Any]):
+        # add 'note' value to metadata if Airflow is version 2.5.0 or later
+        if version.parse(airflow_version) >= version.parse("2.5.0"):
+            self.__metadata_templates["note"] = "{{dag_run.note}}"
         """Enrich metadata with values for lakeFS."""
```

Comment:
I think you misformatted the new code with respect to the PyDoc:

```suggestion
        """Enrich metadata with values for lakeFS."""
        # add 'note' value to metadata if Airflow is version 2.5.0 or later
        if version.parse(airflow_version) >= version.parse("2.5.0"):
            self.__metadata_templates["note"] = "{{dag_run.note}}"
```

Reply:
Another reason to love Python: docstring are _so_ fragile...